### DISCUSSION TITLE
Delete locked out account

### DISCUSSION THREAD

**Discussion by sourabsridhar** [community/community#120944] [Other Feature Feedback, Questions, & Ideas]
Hello,

It looks like I am locked out of my github account @sourabbsridhar (I remember my password, but I have lost my 2FA and can't find recover codes, passkeys, or verified SSH devices). I was able to unlink from my account, but I still have my profile picture and other personal details on that account. Is it possible to delete my locked out account (i.e., @sourabbsridhar)? I have created a new personal github account (@sourabsridhar), and I can use that account for my future use.

Thanks a lot!

**Comment by niquenen** [comment]
You must access to your account to [delete](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-your-personal-account/deleting-your-personal-account) it. The GitHub support team cannot perform this action for you. If the [recovery methods](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials) don't work, you can [unlink the email address](https://docs.github.com/en/authentication/securing-your-account-with-two-factor-authentication-2fa/recovering-your-account-if-you-lose-your-2fa-credentials#unlinking-your-email-address) and use it with your new account.

> If you have forgotten your password and you've lost access to your two-factor authentication credentials, you can start account recovery to regain access to your account. You'll need to verify your identity using a recovery authentication factor, such as an SSH key or previously verified device. __If no recovery methods are available, you can choose to unlink your email address from your account__.

**Reply by joaopa24** [reply]
@niquenen Let's suppose an account has my name and photo, am I still unable to delete it? Isn't it against some policy?

**Reply by niquenen** [reply]
> @niquenen Let's suppose an account has my name and photo, am I still unable to delete it? Isn't it against some policy?

@joknz21, in this case, if someone has stolen your name and photo, you can [report a user profile](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam#reporting-a-user).

A [private information removal request](https://docs.github.com/en/site-policy/content-removal-policies/github-private-information-removal-policy#private-information-removal-requests-are-not-appropriate-for) is not appropriate for a privacy complaint, see: [GitHub Private Information Removal Policy](https://docs.github.com/en/site-policy/content-removal-policies/github-private-information-removal-policy). The fastest solution is to create a ticket and ask the support team or send an email to [support@github.com](mailto:support@github.com).

**Comment by ebndev** [comment]
Hi @sourabsridhar,

This community is managed by GitHub employees, but it is not a support forum. The only place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact).

This is a public forum, and the community is unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours.

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion and any duplicate posts.

Thank you for understanding!
